_layouts/sheafy/tree/default.html

```diff
@@ -9,11 +9,17 @@
 
     <h1 class="post-title">
       {% if page.collection == "nodes" %}
-      <span class="numbering">{{ page.iso_2145 | append: "." }}</span>
+        <span class="numbering">{{ page.clicks | to_numbering }}</span>
+      {% endif %}
+      {% if page.title != page.slug %}
+        {{ page.title }}
+      {% elsif page.genus %}
```

Comment:
Actually it is, but probably not in the way you'd expect -- i.e. the work is working as _I_ expect.

Definition with title, rendered as standalone:

![image](https://user-images.githubusercontent.com/2499655/151437582-e3ec9a7d-6ac7-4173-8112-d1fa99777084.png)

Definition without title, rendered as standalone:

![image](https://user-images.githubusercontent.com/2499655/151437544-15898f23-64f5-4f59-9606-ba5695197977.png)

All this is to avoid showing ugly stuff like `6.1❋a 0015 [0015]` unless unavoidable, without being too verbose, as it's a huge heading.

However, given your comment https://github.com/jonsterling/math/pull/28#discussion_r793912032 the issue is probably void as that requires removing headings from the layout (i.e. removing the problematic code here) and keeping the ones in the sublayout always on.